vit-servicing-station-tests/src/tests/rest/service_version.rs

```diff
@@ -0,0 +1,29 @@
+use crate::common::{
+    data,
+    startup::{db::DbBuilder, server::ServerBootstrapper},
+};
+use assert_fs::TempDir;
+
+#[test]
+pub fn service_version() {
+    let temp_dir = TempDir::new().unwrap();
+    let (hash, token) = data::token();
+
+    let db_path = DbBuilder::new().with_token(token).build(&temp_dir).unwrap();
+    let version = "TestV1".to_string();
```

Comment:
We definitely need to have an agreement about versioning the VITss API and VITss as well (i.e. we will implement frequent tagging in CI for VITss repo), so there will be no need to hardcode the versioning in tests in near perspective.